lib/parsers/asm-parser.ts

```diff
@@ -80,7 +80,7 @@ export class AsmParser extends AsmRegex implements IAsmParser {
     stdInLooking: RegExp;
     endBlock: RegExp;
     blockComments: RegExp;
-    constructor(compilerProps) {
+    constructor(compilerProps?) {
```

Comment:
We can type CompilerProps as PropertyGetter

Reply:
Great point! I'll rebase post jeremy's big compiler thing and update.